MANIFEST.in

```diff
@@ -2,3 +2,4 @@ include README.md
 include LICENSE
 include discretisedfield/util/plotting-style.mplstyle
 recursive-include discretisedfield/tests/test_sample *
+include discretisedfield/html/templates/*.jinja2
```

Comment:
No newline at end of file. Not sure what the convention is, but github is flagging this ;)